dart/pubspec.yaml

```diff
@@ -6,6 +6,7 @@ description: >
 homepage: https://docs.sentry.io/platforms/dart/
```

Comment:
Even though the homepage already linked to the docs, I never happend to click on it because I assumed it leads me to `sentry.io` instead of the docs.